docs/modules/ROOT/pages/api/security.adoc

```diff
@@ -0,0 +1,200 @@
+
+== Initializable
+
+[.contract]
+[[InitializableComponent]]
+=== `++InitializableComponent++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.7.0/src/security/initializable.cairo[{github-icon},role=heading-link]
+
+```javascript
+use openzeppelin::security::InitializableComponent;
+```
+
+Component enabling one-time initialization for contracts.
+
+[.contract-index]
+.Embeddable Implementations
+--
+.InitializableImpl
+
+* xref:#InitializableComponent-is_initialized[`++is_initialized(self)++`]
+--
+
+[.contract-index]
+.Internal Implementations
+--
+.InternalImpl
+
+* xref:#InitializableComponent-initialize[`++initialize(self)++`]
+--
+
+[#InitializableComponent-Embeddable-Functions]
+==== Embeddable Functions
+
+[.contract-item]
+[[InitializableComponent-is_initialized]]
+==== `[.contract-item-name]#++is_initialized++#++(self: @ComponentState)++ → bool` [.item-kind]#external#
+
+Returns whether the contract has been initialized.
+
+[#InitializableComponent-Internal-Functions]
+==== Internal Functions
+
+[.contract-item]
+[[InitializableComponent-initialize]]
+==== `[.contract-item-name]#++initialize++#++(ref self: ComponentState)++` [.item-kind]#internal#
+
+Ensures the calling function can only be called once.
+
+Requirements:
+
+- the contract must not have been initialized before.
+
+== Pausable
+
+[.contract]
+[[PausableComponent]]
+=== `++PausableComponent++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.7.0/src/security/pausable.cairo[{github-icon},role=heading-link]
+
+:Paused: xref:PausableComponent-Paused[Paused]
+:Unpaused: xref:PausableComponent-Unpaused[Unpaused]
+
+```javascript
+use openzeppelin::security::PausableComponent;
+```
+
+Component to implement an emergency stop mechanism.
+
+[.contract-index]
+.Embeddable Implementations
+--
+.PausableImpl
+
+* xref:#PausableComponent-is_paused[`++is_paused(self)++`]
+--
+
+[.contract-index]
+.Internal Implementations
+--
+.InternalImpl
+
+* xref:#PausableComponent-assert_not_paused[`++assert_not_paused(self)++`]
+* xref:#PausableComponent-assert_paused[`++assert_paused(self)++`]
+* xref:#PausableComponent-_pause[`++_pause(self)++`]
+* xref:#PausableComponent-_unpause[`++_unpause(self)++`]
+--
+
+[.contract-index]
+.Events
+--
+* xref:#PausableComponent-Paused[`++Paused(account)++`]
+* xref:#PausableComponent-Unpaused[`++Unpaused(account)++`]
+--
+
+[#PausableComponent-Embeddable-Functions]
+==== Embeddable Functions
+
+[.contract-item]
+[[PausableComponent-is_paused]]
+==== `[.contract-item-name]#++is_paused++#++(self: @ComponentState)++ → bool` [.item-kind]#external#
+
+Returns whether the contract is currently on pause.
+
+[#PausableComponent-Internal-Functions]
+==== Internal Functions
+
+[.contract-item]
+[[PausableComponent-assert_not_paused]]
+==== `[.contract-item-name]#++assert_not_paused++#++(self: @ComponentState)++` [.item-kind]#internal#
+
+Makes a function only callable when the contract is not paused.
```

Comment:
```suggestion
Panics if the contract is paused.
```
I think this is more explicit and clear than "not callable", which might simply mean it does nothing. It's also important to be clear when a function reverts execution.